dbt-bigquery/pyproject.toml

```
@@ -27,7 +27,7 @@ dependencies = [
     "dbt-adapters>=1.7,<2.0",
     # 3.20 introduced pyarrow>=3.0 under the `pandas` extra
     "google-cloud-bigquery[pandas]>=3.0,<4.0",
-    "google-cloud-storage~=2.4",
+    "google-cloud-storage>=2.4",
```

We'd be happy to merge this if you could hard cap this from above on the major. We don't know how the latest will play with older pinned dependencies so it's safer to pin from above on the major.

```suggestion
    "google-cloud-storage>=2.4,<3.0",
```